sdk/src/main/java/com/sygic/travel/sdk/common/responseWrappers/StResponse.kt

```diff
@@ -1,12 +1,12 @@
-package com.sygic.travel.sdk.api.responseWrappers
+package com.sygic.travel.sdk.common.responseWrappers
 
 import com.google.gson.annotations.SerializedName
 
 /**
  * Base of an API response. It's extended by specific respond classes containing the specific data.
  * It's abstract, because of the abstract [.getData] method.
  */
-internal abstract class StResponse {
+abstract class StResponse {
```

Comment:
why not internal?

Reply:
All subclasses of `StResponse` have to be internal too e.g.: Subclass `MediaResponse` have to be internal too, but it cannot because method `getPlaceMedia()` in `PlacesApiClient` interface exposes its return type argument because in interface all methods must be public. And you cannot make the interface internal because Facade must be public and the Facade exposes Service which exposes the interface `PlacesApiClient`